assume/reinforcement_learning/learning_role.py

```diff
@@ -45,6 +45,8 @@ def __init__(
         self.buffer: ReplayBuffer = None
         self.obs_dim = learning_config["observation_dimension"]
         self.act_dim = learning_config["action_dimension"]
+        self.early_stopping_steps = learning_config["early_stopping_steps"]
```

Comment:
these should have some default value set by us to make things a bit easier for the user